sdk/eventhubs/azure_messaging_eventhubs/README.md

```diff
@@ -207,23 +208,24 @@ async fn receive_events() {
         None,
         credential,
         Some(
-            azure_messaging_eventhubs::consumer::ConsumerClientOptions::builder()
-                .with_application_id("receive_lots_of_events")
-                .build(),
+            azure_messaging_eventhubs::consumer::ConsumerClientOptions{
+                application_id: Some("receive_lots_of_events".to_string()),
+                ..Default::default()}
         ),
     );
 
     client.open().await.unwrap();
 
     let event_stream = client
         .receive_events_on_partition(
-            "0",
+            "0".to_string(),
             Some(
-                azure_messaging_eventhubs::consumer::ReceiveOptions::builder()
-                    .with_start_position(azure_messaging_eventhubs::consumer::StartPosition::builder().with_earliest_location().build())
-                    .build(),
-            ),
-        )
+                azure_messaging_eventhubs::consumer::ReceiveOptions{
+                    start_position: Some(azure_messaging_eventhubs::consumer::StartPosition{
+                        location: azure_messaging_eventhubs::consumer::StartLocation::Earliest,
+                        ..Default::default()}),
+                    ..Default::default()},
```

Comment:
Nit: unless the formatter did this, I would put the struct termination on separate lines e.g.,
```suggestion
                        ..Default::default()
                    }),
                    ..Default::default()
                },
```

Must cleaner to read. The "trailing commas" threw me for a sec since you can do that with the "and the rest" operators until I noticed the `}` in there.